scpc/hybridmixed.py

```diff
@@ -22,6 +22,8 @@ class HybridizationPC(PCBase):
     are performed element-local using the Slate language.
     """
 
+    needs_python_pmat = True
+
     @timed_function("HybridInit")
```

Comment:
All these preconditioners get info out of the Python context, so
need a python pmat.